### PR TITLE
CSPL-2200: fix ansible code to not update web.conf when set to "false"

### PR DESCRIPTION
**Pull request by gaurav-splunk**
Fix ansible code to not update web.conf in etc/system/local/web.conf when not explicitly set to true.

### CODE REVIEW

File: roles/splunk_common/tasks/main.yml [hunk 1]
```diff
@@ -130,4 +131,4 @@
 - include_tasks: add_splunk_license.yml
 
 - include_tasks: disable_popups.yml
-  when: "'disable_popups' in splunk and splunk.disable_popups | bool"
\ No newline at end of file
+  when: "'disable_popups' in splunk and splunk.disable_popups | bool"
```
Comment (akondur) [inline roles/splunk_common/tasks/main.yml:134]: Was this change intentional?

**Review by akondur — APPROVED** [review]
Looks good

**Review by kumarajeet — APPROVED** [review]
Approving assuming @akondur comment is validated.

File: roles/splunk_common/tasks/main.yml (hunk 1 above)
Comment (gaurav-splunk) [reply roles/splunk_common/tasks/main.yml:134]: nope I don't think so I made any change in that.

Comment (akondur) [reply roles/splunk_common/tasks/main.yml:134]: Looks like a few extra spaces might have been removed on line 135 and that is fine. Closing the comment thread now.

**Review by alishamayor — APPROVED** [review]
